src/test/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporterTest.java

```diff
@@ -265,15 +265,18 @@ public void shouldCorrectlyRecordGenericAdminEvents() throws IOException {
 
     @Test
     public void shouldCorrectlyRecordResponseDurations() throws IOException {
-        PrometheusExporter.instance().recordRequestDuration(5, "GET");
-        assertGenericMetric("keycloak_request_duration_count", 1, tuple("method", "GET"));
-        assertGenericMetric("keycloak_request_duration_sum", 5, tuple("method", "GET"));
+        PrometheusExporter.instance().recordRequestDuration(5, "GET", "/auth/users");
+        assertGenericMetric("keycloak_request_duration_count", 1,
+            tuple("method", "GET"), tuple("resource", "/auth/users"));
+        assertGenericMetric("keycloak_request_duration_sum", 5,
+            tuple("method", "GET"), tuple("resource", "/auth/users"));
     }
 
     @Test
     public void shouldCorrectlyRecordResponseErrors() throws IOException {
-        PrometheusExporter.instance().recordResponseError(500, "POST");
-        assertGenericMetric("keycloak_response_errors", 1, tuple("code", "500"), tuple("method", "POST"));
+        PrometheusExporter.instance().recordResponseError(500, "POST", "/auth/users");
```

Comment:
can we add a test case that includes user ids to make sure they are not recorded?
(see https://github.com/aerogear/keycloak-metrics-spi/issues/64 )

Reply:
I changed the algorithm and I think this comment is not valid anymore.